### PR TITLE
TRUNK-5968 Add empty changeset to Liquibase update file

### PR DESCRIPTION
**Pull request by WolfSchlegel**
## Description of what I changed
In https://github.com/openmrs/openmrs-core/pull/3728 @BartlomiejRasztabiga found that `api/src/test/java/org/openmrs/liquibase/ChangeLogDetectiveDatabaseIT` fails when a Liquibase update file does not contain any change sets.

The test was fixed by excluding the file from the test and leaving a **todo** in the code base to include the file again at a later stage.

This pull request fixes the issue in a different way by

- adding an empty change set to `api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml`
- introducing a new template for future Liquibase update files 

This approach avoids the **todo** that needs to be resolved by someone in the future.

## Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-5968

## Checklist: I completed these to help reviewers :)
- [ x ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.
- [ x ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)
- [ x ] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
- [ x ] All new and existing **tests passed**.
- [ x ] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: liquibase/README.md [hunk 1]
```diff
@@ -276,6 +282,8 @@ sets that are introduced
 
 Include the new file in `resources/liquibase-update-to-latest-from-1.9.x.xml`, it is used by integration tests (as mentioned above).
 
+The
```
Comment (BartlomiejRasztabiga) [inline liquibase/README.md:285]: Is this a leftover? :)

**Review by BartlomiejRasztabiga — APPROVED** [review]

File: api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml [hunk 2]
```diff
@@ -18,5 +18,12 @@
 	    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
 		http://www.liquibase.org/xml/ns/dbchangelog 
 		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+	<!--
+		The empty change set is needed for integration tests. It can be removed if this file contains other change sets.
+	-->
+	<changeSet id="42ddb873-f064-4418-a6c0-6c09426c4832" author="wschlegel">
```
Comment (dkayiwa) [inline api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml:25]: Isn't it easy for one to forget removing this changeset, and in which case it ends up being released in production? Yet for the test, we do not have this risk, however small it it. 😊

File: liquibase/README.md (hunk 1 above)
Comment (WolfSchlegel) [reply liquibase/README.md:285]: No, `resources/liquibase-update-to-latest-from-1.9.x.xml` is used by `api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java`. That test can probably be refactored or decommissioned but that is not in scope of this ticket.

File: api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml (hunk 2 above)
Comment (WolfSchlegel) [reply api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml:25]: Removing the empty change set is good practice but not strictly needed. It is ok to release it to production as it does not do anything. 

The test on the other hand could slowly become out of date when the todo is not resolved. And people would need to remember to add and remove todos every time new snapshots are created.

File: liquibase/README.md (hunk 1 above)
Comment (dkayiwa) [reply liquibase/README.md:285]: I think by  left over he means `The` on line 285 above. 😊

Comment (BartlomiejRasztabiga) [reply liquibase/README.md:285]: @dkayiwa true! Sorry for the confusion

Comment (WolfSchlegel) [reply liquibase/README.md:285]: Got it...

Comment (WolfSchlegel) [reply liquibase/README.md]: And got rid of it...

**Pull request merged**